src/ui/agents/KivaAgent.ts

```diff
@@ -97,6 +97,7 @@ export default class KivaAgent extends BaseAgent implements IAgent {
     }
 }
 
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
These interfaces could be deleted, too, but this is fine

Reply:
Done